internal/nodes/dequeue_ingestion.go

```diff
@@ -160,12 +160,29 @@ func DequeueIngestion(key string, fullsync bool) {
 
 	// L4Rule CRD processing.
 	if objType == lib.L4Rule {
+
+		// L4 Service
 		svcNames, found := schema.GetParentServices(name, namespace, key)
 		if found && utils.CheckIfNamespaceAccepted(namespace) {
 			for _, svcNSNameKey := range svcNames {
 				handleL4Service(utils.L4LBService+"/"+svcNSNameKey, fullsync)
 			}
 		}
+
+		// L4 Service with shared vip annotation
+		for _, svc := range svcNames {
```

Comment:
This will run for normal L4 services also. Can we make it as if `L4svc have shared annotation` do this else `do this`

Reply:
I have addressed the comment.